ui/app/services/permissions.js

```diff
@@ -35,8 +35,8 @@ const API_PATHS = {
 
 const API_PATHS_TO_ROUTE_PARAMS = {
   'sys/auth': ['vault.cluster.access.methods'],
-  'identity/entities': ['vault.cluster.access.identity', 'entities'],
-  'identity/groups': ['vault.cluster.access.identity', 'groups'],
+  'identity/entity/id/': ['vault.cluster.access.identity', 'entities'],
+  'identity/group/id/': ['vault.cluster.access.identity', 'groups'],
```

Comment:
I don't remember if the trailing slash is significant here - what would happen if the path in the policy is `identity/entity/id*`?

Reply:
the slash is only significant in that the key here (`identity/entity/id/`) must match the corresponding value in `API_PATHS` (which is also currently `identity/entity/id/`). i'll remove  the trailing slash since it would be more flexible and catch various paths like `identity/entity/id*` or `identity/entity/id/*` etc. nice catch!